src/gui/accountmanager.cpp

```diff
@@ -81,9 +82,13 @@ bool AccountManager::restore()
         if (!skipSettingsKeys.contains(settings->group())) {
             if (auto acc = loadAccountHelper(*settings)) {
                 acc->_id = accountId;
-                if (auto accState = AccountState::loadFromSettings(acc, *settings)) {
-                    addAccountState(accState);
-                }
+                auto accState = AccountState::loadFromSettings(acc, *settings);
```

Comment:
I'd prefer if you kept the `if`. Currently loading AccountStates can't fail, but it might in the future. (Otherwise I'd use ENFORCE here, since adding a null AccountState to the list would lead to trouble down the line.)